packages/cli/src/commands/hydrogen/generate/routes.ts

```diff
@@ -0,0 +1,26 @@
+import GenerateRoute from './route.js';
+import Command from '@shopify/cli-kit/node/base-command';
+
+export default class GenerateRoutes extends Command {
+  static description = 'Generate all supported routes.';
+
+  static hidden: true;
+
+  static flags = GenerateRoute.flags;
+
+  async run(): Promise<void> {
+    // @ts-ignore
+    const {flags} = await this.parse(GenerateRoutes);
+
+    await GenerateRoute.run([
```

Comment:
Wouldn't it be easier to refactor the `route` command and extract the logic out of the `run` method? Then we can simply call that function there instead of relaying the flags 🤔

Reply:
Yeah, I see what you mean and agree. Going to merge this for now, but moving forward I think we take that approach as we refactor with more unit tests and be more strict about the separation of concerns.